backend/db/dao.go

```diff
@@ -62,6 +62,26 @@ func CreateUser(request model.User, db *gorm.DB) error {
 	return nil
 }
 
+//NOTE: THIS CANNOT BE USED FOR UPDATING PASSWORDS
+//How would I enforce that in the handler? Also should I not allow you to change the email?
+func UpdateUser(request model.User, db *gorm.DB) error {
+	result := db.Save(request)
+
+	if result.Error != nil {
+		return result.Error
+	}
+
+	return nil
+}
+
+func UpdatePass(request model.PassUpdate, db *gorm.DB) error {
+	result := db.Save(request) //will need to test save function
```

Comment:
I don't think that this will work. Gorm I think try to update a table named pass_update with the info, not the user table. you will probably need to do something like a prepared statement here

Reply:
I made a commit with one option and then another with another option for solving this. Although the latter commit I'm not sure makes as much sense on the front end. I'm not sure how it would look like... The first commit seems like a better way but I don't think I can just call a backend function and have it return like that given how these functions work, even if it was already implimented (I am implimenting it in a seperate branch right now).